torchgeo/datasets/benin_cashews.py

```diff
@@ -410,3 +412,74 @@ def _download(self, api_key: Optional[str] = None) -> None:
         target_archive_path = os.path.join(self.root, self.target_meta["filename"])
         for fn in [image_archive_path, target_archive_path]:
             extract_archive(fn, self.root)
+
+    def plot(
+        self,
+        sample: Dict[str, Tensor],
+        show_titles: bool = True,
+        suptitle: Optional[str] = None,
+    ) -> plt.Figure:
+        """Plot a sample from the dataset.
+
+        This method only plots the first 5 images from the 70 image time series.
```

Comment:
In another PR we decided to have a time index for this kind of thing. Do we want to do the same here for consistency?

Reply:
yes